roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\n' docker"
```

Comment:
@ashcrow I think you might be looking for this: https://docs.ansible.com/ansible/playbooks_advanced_syntax.html#id4

```
!unsafe 'unsafe value'

- name: Determine installed docker version
  command: !unsafe "docker version --format '{{ json.Server}}'"
```
You might be able to use the `{%raw}..{%endraw}` template feature as well.